src/js/health-records/containers/DownloadPage.jsx

```diff
@@ -59,6 +60,23 @@ export class DownloadPage extends React.Component {
     return <AlertBox {...alertProps}/>;
   }
 
+  renderConfirmModal() {
+    const linkOnClick = (e) => {
+      e.preventDefault();
+      this.props.openModal('Are you sure you want to leave this page?', <div>
+        <p>Generating a new health record will replace your most recent download request.</p>
+        <div className="va-modal-actions">
+          <Link to="/" onClick={this.props.closeModal}>
+            <button>Yes, continue</button>
```

Comment:
I think this is considered invalid HTML5. http://stackoverflow.com/a/6393863

Maybe just class the `Link` as `usa-button`?